conf/defaults.ini

```diff
@@ -1057,9 +1057,11 @@ prefix = prod.grafana.%(instance_name)s.
 #################################### Grafana.com integration  ##########################
 [grafana_net]
 url = https://grafana.com
+api_url = https://grafana.com/api
 
 [grafana_com]
 url = https://grafana.com
+api_url = https://grafana.com/api
```

Comment:
I was about to suggest the same but on 2nd thought are these settings meaningful for other users except `grafana_com` so that they need to override them?

Reply:
👍 Yeah, that's a fair point. I'll leave it up to you @sd2k.

Reply:
Hm I added it anyway. It might come in handy for docs internally 🤷